examples/full_dimensional_polytope/full_dimensional_example.cpp

```diff
@@ -0,0 +1,386 @@
+    
+    print_results(n, N.cols(), shift, N, A_full, b_full);
+}
+
+/**
+ * Example 6: Birkhoff polytope
+ * Real-world example: doubly stochastic matrices
+ */
+void example_birkhoff_polytope()
+{
+    print_header("Example 6: Birkhoff Polytope (doubly stochastic matrices)");
+    
+    std::cout << "\nDESCRIPTION:" << std::endl;
+    std::cout << "The Birkhoff polytope B_3 consists of 3x3 doubly stochastic matrices:" << std::endl;
+    std::cout << "  - Each row sums to 1" << std::endl;
+    std::cout << "  - Each column sums to 1" << std::endl;
+    std::cout << "  - All entries are non-negative" << std::endl;
+    std::cout << "This is a 9D space with 6 equality constraints -> 3D polytope" << std::endl;
+    
+    // Generate Birkhoff polytope of size 3
+    Hpolytope birkhoff = generate_birkhoff<Hpolytope>(3);
+    
+    int n = birkhoff.dimension();  // Should be 9 for 3x3 matrices
+    MT A = birkhoff.get_mat();
+    VT b = birkhoff.get_vec();
+    
+    std::cout << "\nOriginal polytope dimension: " << n << std::endl;
+    std::cout << "Number of inequality constraints: " << A.rows() << std::endl;
+    
+    // For demonstration, let's use a simple equality: sum of all entries = 3
+    MT Aeq_dense = MT::Ones(1, n);
+    SpMT Aeq = dense_to_sparse(Aeq_dense);
+    
+    VT beq(1);
+    beq << 3;  // Sum of all entries in a 3x3 doubly stochastic matrix
+    
+    std::cout << "\nUsing simple equality constraint: sum of all entries = 3" << std::endl;
+    
+    auto [A_full, b_full, shift, N] = compute_full_dimensional_polytope(Aeq, beq, A, b);
+    
+    print_results(n, N.cols(), shift, N, A_full, b_full);
+}
+
+int main()
+{
+    std::cout << std::string(70, '=') << std::endl;
+    std::cout << "  FULL DIMENSIONAL POLYTOPE EXAMPLES" << std::endl;
+    std::cout << "  Transforming lower-dimensional polytopes to full dimension" << std::endl;
+    std::cout << std::string(70, '=') << std::endl;
+    
+    std::cout << "\nOVERVIEW:" << std::endl;
+    std::cout << "These examples demonstrate the compute_full_dimensional_polytope function," << std::endl;
+    std::cout << "which transforms a polytope defined by:" << std::endl;
+    std::cout << "  - Equality constraints: Aeq * x = beq" << std::endl;
+    std::cout << "  - Inequality constraints: A * x <= b" << std::endl;
+    std::cout << "into a full-dimensional polytope: A_full * y <= b_full" << std::endl;
+    std::cout << "\nThe transformation provides:" << std::endl;
+    std::cout << "  - shift: A point satisfying the equality constraints" << std::endl;
+    std::cout << "  - N: Nullspace basis (orthonormal columns)" << std::endl;
+    std::cout << "  - Mapping: x = shift + N * y" << std::endl;
+    
+    std::cout << "\n" << std::string(70, '-') << std::endl;
+    std::cout << "Press Enter to run each example..." << std::endl;
+    std::cin.get();
+    
+    // Run all examples
+    example_canonical_simplex();
+    std::cout << "\nPress Enter for next example...";
```

Comment:
There are no similar cases in other examples I think it is simpler without user interference